daemon/api_test.go

```diff
@@ -701,48 +696,45 @@ func (s *apiSuite) TestListIncludesAll(c *check.C) {
 	found := 0
 
 	ast.Inspect(f, func(n ast.Node) bool {
-		switch v := n.(type) {
-		case *ast.ValueSpec:
-			found += len(v.Values)
-			return false
+		// a ValueSpec is a constant or variable declaration
+		vs, ok := n.(*ast.ValueSpec)
+		if !ok {
+			return true
+		}
+		// foo, bar = Command{}, Command{} -> two v.Values
```

Comment:
<img width="699" alt="zrzut ekranu 2018-02-16 o 12 42 28" src="https://user-images.githubusercontent.com/784262/36306180-e8b2da58-1316-11e8-8cc4-8ac8cc4f9d18.png">

Reply:
I've purposely kept it as dumb as possible though (or attempted to)